AUTHORS

```diff
@@ -0,0 +1,10 @@
+# This is the list of http-mock-adapter authors for copyright purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder. To see the full list
+# of contributors, see the revision history in source control.
+Lomsa
+Dato Barbakadze
+Giorgi Beriashvili
+Luka Giorgadze
+Natalie Alibegashvili
```

Comment:
ხომ არ ჯობია აქ სახელის კლიკზე გითაბის ექაუნთზე გადავიდეს?

Reply:
I don't know, this is is an informal standard used throughout many GitHub projects. It's not a markdown file.

Many open-source projects write name and surname and be done with it. Some [also add emails for contacting purposes](https://github.com/flutter/flutter/blob/master/AUTHORS).

Reply:
Ah, okay got it. You are good man.

Reply:
We could specify the GitHub links in another way, most probably inside the **README.md** file.

Reply:
@LukaGiorgadze should I include the emails inside **AUTHORS**? and GitHub links inside the **README.md**?

Reply:
No, i guess full name is enough for now. Everyone can see contributors here.